static/js/cookie-proxy.js

```diff
@@ -0,0 +1,3 @@
+var cpNs = require('exports-loader?cpNs!cookie-policy/build/js/cookie-policy');
```

Comment:
This file is not currently set up as a module, so we use the special loader to expose the global variable that we need.